ui/app/src/models/Publishing.ts

```diff
@@ -26,9 +28,7 @@ export interface Package {
   comment: string;
 }
 
-export interface Selected {
-  [keys: string]: boolean;
-}
+export type Selected = LookupTable<boolean>;
```

Comment:
This is being used in multiple parts, but updated it since it is basically a LookupTable